web/src/components/molecules/Asset/Asset/AssetBody/previewTypeSelect.tsx

```diff
@@ -20,14 +15,14 @@ type Props = {
 
 export const PreviewTypeSelect: React.FC<Props> = ({ onTypeChange, style, value }) => {
   const previewTypeList = [
-    { id: 1, name: "PNG/JPEG/TIFF/SVG", value: PreviewType.Image },
+    { id: 1, name: "PNG/JPEG/TIFF/SVG", value: "IMAGE" },
```

Comment:
Can you type this prevIewTypeList? value needs to be of PreviewType and I think its better to type that.
```
type PreviewTypeListItem = {
id: number,
name: string,
value PreviewItem
}
```

Reply:
Done!